main-app/app/scripts/providers/proxy/proxy.js

```diff
@@ -0,0 +1,122 @@
+(function () {
+    'use strict';
+    angular.module("Game.Proxy")
+        .factory('Proxy', ['$http', function($http) {
+
+            var GameProxy = function() {
+                var me = this;
```

Comment:
Fix in later iteration - the proxy is doing too much - it should just communicate with the server.
You might want to create a provider to map the API response to a more useful object (such as a pukka game-model).

The API responds with the same message format irrespective of whether you've called NewGame or MakeMove - meaning you should have only one successful response handler.